### DISCUSSION TITLE
怎样隐藏loading

### DISCUSSION THREAD

**Discussion by yiroon** [bytedance/xgplayer#817] [Q&A]
controls:false,可以隐藏控件，那么怎样隐藏loading和重播按钮？

**Comment by gemxx** [comment]
参考 [`ignores`配置](https://v2.h5player.bytedance.com/config/#%E5%85%B3%E9%97%AD%E5%86%85%E7%BD%AE%E6%8E%A7%E4%BB%B6)

```
new Player({
  el:document.querySelector('#mse'),
  url: 'video_url',
  ignores: ['replay', 'loading']
});
```

**Reply by yiroon** [reply]
并没效果

**Reply by gemxx** [reply]
具体描述下效果，以及贴一下您的配置。最好弄一个线上demo，比如使用 https://codesandbox.io/s/
